### PR TITLE
Merging Paul's changes for Kubernetes.General probe

### PR DESCRIPTION
**Pull request by lcarrazana**
- Consolidating scenarios, only leaving check for K8s dashboard

Relates to #265

### CODE REVIEW

File: service_packs/kubernetes/general/general.go [hunk 1]
```diff
@@ -170,7 +41,7 @@ func (s *scenarioState) theKubernetesWebUIIsDisabled() error {
 	} else {
 		//a "pass" is the absence of a "kubernetes-dashboard" pod
 		for _, v := range pl.Items {
-			if strings.HasPrefix(v.Name, "kubernetes-dashboard") {
+			if strings.HasPrefix(v.Name, "kubernetes-dashboard") { // TODO: k8s dashboard pod name should come from config file, in case it changes in k8s future release
```
Comment (iantivey) [inline service_packs/kubernetes/general/general.go:44]: Agree. Let's set the default value to "Kubernetes-dashboard".

**Comment by iantivey** [comment]
Was the "ensure security contexts are enforced" test removed?  I can't see it in the file diff.

**Comment by lcarrazana** [comment]
Yes, `ensure security contexts are enforced` test was removed.
See new simplified version of general probe in Paul's fork [here](https://github.com/paulm-jones/probr/blob/kube-feature-runthrough/service_packs/kubernetes/general/general.feature)

File: service_packs/kubernetes/general/general.go (hunk 1 above)
Comment (lcarrazana) [reply service_packs/kubernetes/general/general.go:44]: https://github.com/citihub/probr/issues/308

- [ ] TODO

**Comment by lcarrazana** [comment]
- [ ] Resolve conflicts

**Comment by lcarrazana** [comment]
Proposed changes to General probe were documented in #330 and #331 so that further discussion and approval can happen.
This PR is being closed without further actions.

**Pull request closed without merging**